src/algebra/group_power/lemmas.lean

```diff
@@ -524,7 +524,7 @@ theorem nat.cast_le_pow_sub_div_sub {K : Type*} [linear_ordered_field K] {a : K}
   (n : ℕ) :
   (n : K) ≤ (a ^ n - 1) / (a - 1) :=
 (le_div_iff (sub_pos.2 H)).2 $ le_sub_left_of_add_le $
-  one_add_mul_sub_le_pow ((neg_le_self $ @zero_le_one K _).trans H.le) _
+  one_add_mul_sub_le_pow ((neg_le_self $ @zero_le_one K _ _ _ _).trans H.le) _
```

Comment:
```suggestion
  one_add_mul_sub_le_pow ((neg_le_self zero_le_one).trans H.le) _
```
I don't think that you need the `@`.

Reply:
This PR went through two revisions. I added a few `@` on the first revision which might possibly not be needed anymore. I'll take a look.